content/en/docs/test.md

```diff
@@ -369,9 +369,13 @@ Notes catch the reader's attention without a sense of urgency.
 
 You can have multiple paragraphs and block-level elements inside an admonition.
 
-| Or | a | table |
+| Or | | Even |
+| :---------------- | ----------------: | --------------: |
+|   A    | |   Table    |
```

Comment:
```suggestion
```

Reply:
@Rishit-dagli, If we delete the table's body the result won't have the table format. In fact, we will have *Or* *Even* which could be misleading for some users. See the comment above from @Sea-n.